### DISCUSSION TITLE
Using addGeneIntegrationMatrix across conditions with matched scRNA/ATAC-seq data

### DISCUSSION THREAD

**Discussion by rkelly712** [GreenleafLab/ArchR#1977] [Questions / Documentation]
Hi All,
I'm just looking for some clarification on the addGeneIntegrationMatrix function and its use across conditions.

I'm attempting to add scRNA-seq cell labels to matched scATAC-seq data following [chapter 8](https://www.archrproject.com/bookdown/defining-cluster-identity-with-scrna-seq.html) of the tutorial. 
My scRNA-seq and scATAC-seq datasets were collected separately but are derived from the same condition. 
I'd like to conduct integration in a condition-specific manner (i.e., annotations from the control RNA only integrating/annotating to the control ATAC-seq data and the same for my knockout data). I'm reasonably sure I have the right idea but I'd just like some clarification.

I believe I can define the condition by cluster in the ArchR cellColData as you've described previously [#696](https://github.com/GreenleafLab/ArchR/discussions/696) but for the scRNA-seq, I'm a bit lost. I thought having a column defined as something like "AnnotationByCondition" and passing that to groupRNA would be sufficient but I don't think it ensures condition-to-condition integration. I'm just wondering what the best workaround would be.

Any information is appreciated. 

Thanks in advance and to the authors for the great package! ArchR has made my scATAC-seq work much easier than before!

Ryan

**Comment by Adam-JJJJJ** [comment]
Hello, do you have any ideas for this?

Similarly, I have 7 cell types, each with two conditions: Cell_type1_Control, Cell_type1_Treatment, and so on. For downstream analysis, I want to identify cell_type_condition-specific peak-to-gene (p2g) correlations for Cell_type1_Treat. This would indicate that certain loops exhibit higher correlation specifically in Cell_type1_Treatment, suggesting they may serve unique functional roles.

Currently, I performed addGeneIntegrationMatrix with constraints within the same cell type in both single-cell RNA (scRNA) and single-cell ATAC (scATAC), resulting in 7 pairs in the "groupList". However, I found that some cells labeled as Cell_type1_Control in scATAC have integrated with Cell_type1_Treatment in scRNA. Therefore, I am wondering whether I should impose constraints on Cell_type1_Condition across both scRNA and scATAC datasets.

I'd appreciate any insights you might have.
